.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   Build-k-NN:
     strategy:
       matrix:
-        java: [14]
+        java: [11, 14]
```

Comment:
Do we still need to run against 14?

Reply:
My understanding is - yes, ideally we need to support all those JDKs, as Daniel has stated in issue requirements - "Add JDK ... to the CI matrix"